src/services/transactions.rs

```diff
@@ -45,9 +45,14 @@ pub fn get_all_transactions(safe_address: &String) -> Result<Vec<ServiceTransact
     let body = reqwest::blocking::get(url)?.text()?;
     println!("request URL: {}", &url_string);
     println!("{:#?}", body);
-    let transactions: Page<TransactionDto> = serde_json::from_str(&body)?;
-    let transactions: Vec<ServiceTransaction> = transactions.results.into_iter()
-        .flat_map(|transaction| transaction.to_service_transaction())
+    let backend_transactions: Page<TransactionDto> = serde_json::from_str(&body)?;
+    let service_transactions: Vec<ServiceTransaction> = backend_transactions.results.into_iter()
+        .flat_map(|transaction| transaction.to_service_transaction().unwrap())
         .collect();
-    Ok(transactions)
+    Ok(Page {
+        count: service_transactions.len(),
+        next: backend_transactions.next,
```

Comment:
https://github.com/rmeissner/safe-merc-service/blob/master/src/renderer/overview.rs#L44

Reply:
Will address in #32